<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageInfo</class>
 <widget class="QDockWidget" name="ImageInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Info</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label_name">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_gameid">
      <property name="text">
       <string>Game ID</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_country">
      <property name="text">
       <string>Country</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_bannertext">
      <property name="text">
       <string>Banner</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="line_name">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Mario Kart Double Dash!</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="line_gameid">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>GM4P01</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="line_country">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>EUROPE</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_banner">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
